themes.go

```diff
@@ -0,0 +1,115 @@
+package main
+
+// Themes return the list of themes.
+var Themes = map[string]Theme{
+	"TokyoNight-Storm": {
```

Comment:
@caarlos0 Thinking about this a bit more, I think it would be a little bit of a pain to store all these themes and maintain this list. This project seems to have all of the themes stored as JSON in a format very similar to what we want.

https://windowsterminalthemes.dev

I wonder if we can use this to piggyback off of to have a ton of themes without needing to maintain all of them ourselves. We could also standardize on a different base16 format (like alacritty color themes) and have all of them be generated or fetched from that.

Reply:
good idea, no cattpuccin theme there though, will pr it and change here


https://github.com/atomcorp/themes/blob/master/themes.json 

we can probably embed this file

Reply:
Nice, yeah sounds like a plan! Thank you!

Reply:
Also regarding theme names I think spaces are the way to go (rather than dashes):

```
Set Theme "Catpuccin Mocha"
```

Reply:
https://github.com/atomcorp/themes/pull/86

Reply:
@maaslalani just pushed a first version of all that, catppuccin is not there yet thougn, need to wait for them to merge

Reply:
Thank you so much @caarlos0, this is incredible! This is going to be much more maintainable.

Reply:
totally yes

Reply:
thank you for the idea of that repo, didnt know it :D